site/docs/v1/tech/apis/_form-field-request-body.adoc

```diff
@@ -0,0 +1,77 @@
+==== Request Body
+
+[.api]
+
+[field]#field.confirm# [type]#[Boolean]# [optional]#Optional#::
+Determines if the `control` field should be verified. If `true`, a confirmation field is included.
+
+[field]#field.consentId# [type]#[UUID]# [optional]#Optional#::
+The Id of an existing link:/docs/v1/tech/apis/consent[Consent].
+
+[field]#field.control# [type]#[String]# [optional]#Optional#::
+The Form Field control. The possible values are:
++
+* `checkbox`
+* `number`
+* `password`
+* `radio`
+* `select`
+* `textarea`
+* `text`
+
+[field]#field.data# [type]#[Object]# [optional]#Optional#::
+An object that can hold any information about the Form Field that should be persisted.
+
+[field]#field.description# [type]#[String]# [optional]#Optional#::
+A description of the Form Field.
+
+[field]#field.key# [type]#[String]# [optional]#Optional#::
+The key is the path to the value in the `user` or `registration` object.
+The possible values include:
++
+* `registration.username`
+* `user.birthDate`
+* `user.email`
+* `user.firstname`
+* `user.fullName`
+* `user.lastName`
+* `user.middleName`
+* `user.mobilePhone`
+* `user.password`
+* `user.username`
+* `registration.data.*`
+* `user.data.*`
+
+[field]#field.name# [type]#[String]# [required]#Required#::
+The unique name of the Form Field.
+
+[field]#field.options# [type]#[Array<String>]# [optional]#Optional#::
+A list of options that are applied to `checkbox`, `radio`, `select`, or `textarea` controls.
+
+[field]#field.required# [type]#[Boolean]# [optional]#Optional#::
+Determines if a value is required for the Form Field.
+
+[field]#field.type# [type]#[String]# [optional]#Optional#::
+The data type used to store the value in FusionAuth. The possible values are:
++
+* `bool`
+* `consent`
+* `date`
+* `email`
+* `number`
+* `string`
+
+[field]#field.validator# [type]#[Object]# [optional]#Optional#::
+When enabled, user input will be evaluated against the `expression`.
```

Comment:
Since this is just two fields deep and we already document both fields, we can probably skip this one.